### DISCUSSION TITLE
call Python script variables in Yaml script

### DISCUSSION THREAD

**Discussion by sesa659920** [community/community#30349] [Actions]
I have prepared python script and printing output values in json format. I need to call that values in github action yaml workflow.

My main assumption is, once python script successfully executed then need to print those output values will displays to  teamschannel notification by using github action workflow.

**Comment by ViacheslavKudinov** [comment]
Hi @sesa659920 
Probably, you can set GHA env vars from Python script using this syntax https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable
